src/routes/docs/references/[version]/models/[model]/+page.server.ts

```diff
@@ -8,8 +8,8 @@ type Model = {
         name: string;
         type: string;
         description: string;
-        example: string | boolean | number | object | Array<unknown>;
-        items?: Array<string>;
+        items?: Array<any>;
+        relatedModels?: string;
```

Comment:
We don't use `example` anymore since we split out the recursive example generation into specs.ts